zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -79,8 +79,8 @@ angular.module('zeppelinWebApp')
   var angularObjectRegistry = {};
 
   var editorModes = {
-    'ace/mode/scala': /^%spark/,
-    'ace/mode/python': /^%pyspark/,
+    'ace/mode/python': /^%(spark\.)?pyspark\s*$/,
+    'ace/mode/scala': /^%spark\s*$/,
```

Comment:
I agree with you about creating a new interpreter group and thanks for pointing that out. 
I'll fix it.
and Regarding the running paragraph without "%spark" when it's default is no problem to work.